packages/camera/camera_android_camerax/pubspec.yaml

```diff
@@ -0,0 +1,73 @@
+name: camera_android_camerax
+description: A new Flutter plugin project.
+repository: https://github.com/flutter/plugins/tree/main/packages/camera/camera_android_camerax
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
+version: 0.0.1
+homepage:
+
```

Comment:
I think this should include a `publish_to: 'none'`. Are you intending on publishing the plugin through development to secure the name?

Reply:
No, I didn't want to publish this. Should the version just be `NEXT` then until it's published?

Reply:
Yea, setting it to `NEXT` would be a good idea.